pcsx2/DEV9/DEV9.cpp

```diff
@@ -1076,3 +1083,74 @@ void DEV9setLogDir(const char* dir)
 	// Get the path to the log directory.
 	s_strLogPath = (dir == NULL) ? "logs" : dir;
 }
+
+void ApplyConfigIfRunning(Config oldConfig)
+{
+	if (!isRunning)
+		return;
+
+	//Eth
+	if (config.ethEnable)
+	{
+		if (oldConfig.ethEnable)
+		{
+			//Reload Net if adapter changed
+			if (strcmp(oldConfig.Eth, config.Eth) != 0)
+			{
+				TermNet();
+				InitNet();
+			}
+		}
+		else
+			InitNet();
+	}
+	else if (oldConfig.ethEnable)
+		TermNet();
+
+		//Hdd
+		//Hdd Validate Path
```

Comment:
Why two comments here?

Reply:
THe `//Hdd` comment is supposed to indicate that this block of code deals with hdd, the following `Hdd Validate Path` is supposed to refer to the immediately following if statements (up to the `//Hdd Compare with old config` comment)